.fastlane/Fastfile

```diff
@@ -4,14 +4,74 @@ default_platform :ios
 
 platform :ios do
   before_all do
-    xcversion(version: "9.2")
+    setup_circle_ci
```

Comment:
Turns out this line is important: https://github.com/circleci/circleci-docs/pull/2258